### PR TITLE
Refactor:USE PARAM instead of GlobalV

### PR DESCRIPTION
**Pull request by A-006**
### Unit Tests and/or Case Tests for my changes
- change the CMakeLists for CAL_FORCE，FORCE_THR，PRESSURE，OUT_LEVEL，relax_new，use_paw，double_grid ，fixed_atoms，SCF_NMAX

### What's changed?
- Change the source file for CAL_FORCE，FORCE_THR，PRESSURE，OUT_LEVEL，relax_new，use_paw，double_grid ，fixed_atoms，SCF_NMAX in GlobalV
- delete them in the global_varibale.h

### CODE REVIEW

File: source/module_io/read_input_item_other.cpp [hunk 1]
```diff
@@ -426,12 +426,6 @@ void ReadInput::item_others()
         read_sync_int(input.nurse);
         this->add_item(item);
     }
-    {
-        Input_Item item("colour");
-        item.annotation = "for coders, make their live colourful";
-        read_sync_bool(input.colour);
-        this->add_item(item);
-    }
```
Comment (Qianruipku) [inline source/module_io/read_input_item_other.cpp:434]: Please remove colour in input-main.md

File: source/module_parameter/input_parameter.h [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #ifndef INPUT_PARAMETER_H
 #define INPUT_PARAMETER_H
+#include "module_parameter/parameter.h"
```
Comment (Qianruipku) [inline source/module_parameter/input_parameter.h]: delete it

**Review by Qianruipku — CHANGES_REQUESTED** [review]

File: source/module_io/read_input_item_other.cpp (hunk 1 above)
Comment (A-006) [reply source/module_io/read_input_item_other.cpp:434]: the problem has been finished

File: source/module_parameter/input_parameter.h (hunk 2 above)
Comment (A-006) [reply source/module_parameter/input_parameter.h]: The problem has been finished

**Review by Qianruipku — APPROVED** [review]

**Pull request merged**